tests/subsys/audio_module/src/common.c

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/fff.h>
+
+#include "common.h"
+#include "audio_module/audio_module.h"
+
+const char *test_instance_name = "Test instance";
```

Comment:
These three are constant, hence the names should be all CAPS.

Reply:
Sorry, should the test_instance_name or Test instance be all CAPS?

Reply:
The first, it should be TEST_INSTANCE_NAME